compiler/util/astlocs.cpp

```diff
@@ -70,6 +70,10 @@ astlocMarker::~astlocMarker() {
 //   not in a function in beginning with chpl__
```

Comment:
While you're here, should this be `not in a function beginning with ...`?

Reply:
I'm having trouble understanding what you are asking? It seems to already say that.

Reply:
There's an extra `in` in there.

Reply:
Gotcha, thanks. Fixed.